engine/src/main/scala/cromwell/engine/workflow/tokens/JobExecutionTokenDispenserActor.scala

```diff
@@ -74,11 +75,14 @@ class JobExecutionTokenDispenserActor(override val serviceRegistryActor: ActorRe
         tokenAssignments = tokenAssignments + (actor -> lease)
         incrementJob("Started")
         actor ! JobExecutionTokenDispensed
-      // Only one token per actor, if you've already got one, you don't get this token !
-      case LeasedActor(_, lease) => lease.release()
+      // Only one token per actor, so if you've already got one, we don't need to use this new one:
+      case LeasedActor(actor, lease) =>
+        actor ! JobExecutionTokenDispensed
+        lease.release()
```

Comment:
what's this change about?

Reply:
Before: When an actor which already has a lease asks for another one, remove it from the queue and leave it hanging.
Now: When an actor which already has a lease asks for another one, remove it from the queue and resend the original `JobExecutionTokenDispensed` message.

Since this is almost certainly a sign of programmer error, I could probably add an error message alongside this